### PR TITLE
AM2R: Tester rebalance

### PR DESCRIPTION
**Pull request by JeffGainsNGames**
Main goal was to fix possibility of a trickless SJ-less fight with enough E-tanks.

Actual Health values still likely need adjusting.

### CODE REVIEW

File: randovania/games/am2r/logic_database/The Tower.txt [hunk 1]
```diff
@@ -929,20 +929,27 @@ Extra - minimap_data: [{'x': 33, 'y': 34}, {'x': 33, 'y': 35}, {'x': 33, 'y': 36
   > Event - Tester
       All of the following:
           Any of the following:
-              # Energy requirements
+              # Weapon Requirements
+              Missiles or Plasma Beam or Power Bombs or Super Missiles
+              Combat (Advanced) and Can Use Charged Bomb Jump
+              Combat (Hypermode) and Can Use Bombs
+              Combat (Expert) and Damage Boost (Advanced) and Knowledge (Advanced) and Normal Damage ≥ 299
```
Comment (Miepee) [inline randovania/games/am2r/logic_database/The Tower.txt:936]: could use a comment that you can circumvent the outer shell by damage boosting into it and shooting it from the inside

File: randovania/games/am2r/logic_database/The Tower.txt [hunk 2]
```diff
@@ -929,20 +929,27 @@ Extra - minimap_data: [{'x': 33, 'y': 34}, {'x': 33, 'y': 35}, {'x': 33, 'y': 36
   > Event - Tester
       All of the following:
           Any of the following:
-              # Energy requirements
+              # Weapon Requirements
+              Missiles or Plasma Beam or Power Bombs or Super Missiles
```
Comment (Miepee) [inline randovania/games/am2r/logic_database/The Tower.txt:933]: why plasma beam, does it circumvent its shell?

File: randovania/games/am2r/logic_database/The Tower.txt [hunk 3]
```diff
@@ -929,20 +929,27 @@ Extra - minimap_data: [{'x': 33, 'y': 34}, {'x': 33, 'y': 35}, {'x': 33, 'y': 36
...
               All of the following:
-                  # With Space jump
+                  # With Space Jump
                   Space Jump
-                  Combat (Advanced) or Normal Damage ≥ 330
+                  Any of the following:
+                      Normal Damage ≥ 330
+                      Combat (Beginner) and Normal Damage ≥ 200
+                      Combat (Intermediate) and Normal Damage ≥ 150
+                      Combat (Advanced) and Normal Damage ≥ 100
```
Comment (Miepee) [inline randovania/games/am2r/logic_database/The Tower.txt:946]: no expert with 65 health?

File: randovania/games/am2r/logic_database/The Tower.txt (hunk 2 above)
Comment (JeffGainsNGames) [reply randovania/games/am2r/logic_database/The Tower.txt:933]: Plasma allows the player to damage Testers life points directly. If you have plasma you can just aim up and spam beam shots.

[the remark this answers is not in the document]
Comment (JeffGainsNGames) [reply randovania/games/am2r/logic_database/The Tower.txt:933]: Leftover from previous requirements, agree though. Should we include enough to break ~2 shell pieces?

[the remark this answers is not in the document]
Comment (Miepee) [reply randovania/games/am2r/logic_database/The Tower.txt:933]: yeah sounds good

[the remark this answers is not in the document]
Comment (JeffGainsNGames) [reply randovania/games/am2r/logic_database/The Tower.txt]: 5 supers per piece, but you do get drops from breaking both the shell and the weapons.
image

**Comment by JeffGainsNGames** [comment]
@Miepee Do you think plasma should be locked behind knowledge beginner since you don't encounter this in vanilla?

**Comment by Miepee** [comment]
Considering i also just asked about it: knowledge beginner does seem fitting, yeah.

**Review by Miepee — APPROVED** [review]

**Pull request merged**